### DISCUSSION TITLE
Output image raw data directly using child_process.spawn

### DISCUSSION THREAD

**Discussion by hemengke1997** [microsoft/vscode-discussions#999] [Extension Development QnA]
I'm trying to output user's image in clipboard by using [Clipboard](https://github.com/Slackadays/Clipboard), the core code is:

```ts
spawn('cb', ['paste'], { stdio: 'inherit' })
```

In pure node env, it works well, the image raw data output to cwd fs system directly.

But same code run in vscode env, I got a buffer(I'm not sure) instead of image raw data, I mean, no image file output, seems that vscode has intercept the output

<img width="730" alt="image" src="https://github.com/microsoft/vscode-discussions/assets/49073282/5d373402-43c7-4013-8588-f09eca6ea526">

Is there any approach to control vscode's behavior? I hope image in clipboard output to file system directly

[Related Discussion](https://github.com/Slackadays/Clipboard/discussions/186#discussioncomment-8014870)

Thank you!

**Comment by isidorn** [comment]
Sorry but I do not have useful context to share. Let's see if someone from the community can help.
